core/Command/User/Info.php

```diff
@@ -68,21 +74,31 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 	 * @param IUser $user
 	 * @return array
 	 */
-	protected function getStorageInfo(IUser $user): array {
+	protected function getStorageInfo(IUser $user, InputInterface $input): array {
 		\OC_Util::tearDownFS();
 		\OC_Util::setupFS($user->getUID());
 		try {
 			$storage = \OC_Helper::getStorageInfo('/');
 		} catch (\OCP\Files\NotFoundException $e) {
 			return [];
 		}
-		return [
-			'free' => $storage['free'],
-			'used' => $storage['used'],
-			'total' => $storage['total'],
-			'relative' => $storage['relative'],
-			'quota' => $storage['quota'],
-		];
+		if ($input->getOption('human')) {
+			return [
+				'free' => \OC_Helper::humanFileSize($storage['free']),
+				'used' => \OC_Helper::humanFileSize($storage['used']),
+				'total' => \OC_Helper::humanFileSize($storage['total']),
+				'relative' => $storage['relative'] . '%',
+				'quota' => ($storage['quota'] >= 0) ? \OC_Helper::humanFileSize($storage['quota']) : $storage['quota'],
```

Comment:
```suggestion
				'free' => Util::humanFileSize($storage['free']),
				'used' => Util::humanFileSize($storage['used']),
				'total' => Util::humanFileSize($storage['total']),
				'relative' => $storage['relative'] . '%',
				'quota' => ($storage['quota'] >= 0) ? Util::humanFileSize($storage['quota']) : $storage['quota'],
```

And add a use `\OCP\Util;` to the top. `OC_*` classes are deprecated.